### PR TITLE
added new test checkWomanTopsJacketsLink

### PR DESCRIPTION
**Pull request by EKlipert**
[Jira link](https://bugbusters07.atlassian.net/browse/TC-1?atlOrigin=eyJpIjoiYWIzYTU5OWIxNDY2NDM1NWIwYmNkZmFjZDRhNGYyOGIiLCJwIjoiaiJ9)

### CODE REVIEW

File: cypress/e2e/checkWomanTopsJacketsLink.cy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  it("TC0001_Check the link for Woman/Tops/Jackets", () => {
+    homePage
+      .hoverWomanMainMenuLink()
+      .hoverWomanTopsLink()
+      .clickWomanJacketsLink()
+      .getJacketsTitle()
+      .should("be.visible")
+      .and("have.text", "Jackets");
+
+    //cy.get("#ui-id-4").trigger('mouseover');
```
Comment (AnnieZ1207) [inline cypress/e2e/checkWomanTopsJacketsLink.cy.js:21]: why we need this commented line?

File: cypress/e2e/checkWomanTopsJacketsLink.cy.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  });
+
+  it("TC0001_Check the link for Woman/Tops/Jackets", () => {
+    homePage
+      .hoverWomanMainMenuLink()
+      .hoverWomanTopsLink()
+      .clickWomanJacketsLink()
+      .getJacketsTitle()
+      .should("be.visible")
+      .and("have.text", "Jackets");
```
Comment (AnnieZ1207) [inline cypress/e2e/checkWomanTopsJacketsLink.cy.js:19]: how about we move  page tittle to the fixtures?

File: cypress/e2e/checkWomanTopsJacketsLink.cy.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// <reference types="cypress"/>
+
+import HomePage from "../pageObjects/HomePage";
+
+describe("checkWomanTopsJacketsLink", () => {
+  const homePage = new HomePage();
+  
+  beforeEach(() => {
+    cy.visit("https://magento.softwaretestingboard.com/");
```
Comment (AnnieZ1207) [inline cypress/e2e/checkWomanTopsJacketsLink.cy.js:9]: i merged main into this branch so now we able to use baseUrl

Comment (EKlipert) [reply cypress/e2e/checkWomanTopsJacketsLink.cy.js:9]: Thanks, fixed

File: cypress/e2e/checkWomanTopsJacketsLink.cy.js (hunk 2 above)
Comment (EKlipert) [reply cypress/e2e/checkWomanTopsJacketsLink.cy.js:19]: Do not need it, fixed

File: cypress/e2e/checkWomanTopsJacketsLink.cy.js (hunk 1 above)
Comment (EKlipert) [reply cypress/e2e/checkWomanTopsJacketsLink.cy.js:21]: Don't need it, fixed

**Review by aliopik — APPROVED** [review]

**Pull request merged**